lib/rucio/daemons/conveyor/common.py

```diff
@@ -178,7 +178,7 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, fts_source_str
                 'selection_strategy': fts_source_strategy,
                 'request_type': transfer['file_metadata'].get('request_type', None),
                 'activity': str(transfer['file_metadata']['activity'])}
-        if file['metadata'].get('verify_checksum', True):
+        if verify_checksum != 'none':
```

Comment:
` != 'none'` looks weird? Shouldn't this be either "is not None' (Python) or " != 'null' " (JSON) ?

Reply:
The options are 'both', 'source', 'destination', 'none' 
See https://github.com/rucio/rucio/blob/aff3b74312bd6e5bc5839cca06c1f221aa340712/lib/rucio/core/transfer.py#L662
I know, it's a bit weird, but these are strings given to FTS.